pkg/machine/e2e/basic_test.go

```diff
@@ -84,8 +84,8 @@ var _ = Describe("run basic podman commands", func() {
 		Expect(stopSession).To(Exit(0))
 
 		// gxproxy should exit after machine is stopped
-		_, err = exec.Command("pgrep", "gvproxy").Output()
-		Expect(err).To(HaveOccurred())
+		out, _ = pgrep("gvproxy")
+		Expect(strings.Contains(out, "gvproxy")).To(BeFalse())
```

Comment:
Note kind of error matches create useless error mesages, `Expected true to be false` means nothing to a log reader.

This should really be `Expect(out).To(ContainSubstring("gvproxy"))`

Also shouldn't this still check for the error?

Reply:
in this case, pgrep's behavior is that it would return an error because it did not find it.  long term, an alternative approach might be better